bot/exts/moderation/infraction/superstarify.py

```diff
@@ -152,15 +152,26 @@ async def action() -> None:
         old_nick = escape_markdown(old_nick)
         forced_nick = escape_markdown(forced_nick)
 
-        superstar_reason = f"Your nickname didn't comply with our [nickname policy]({NICKNAME_POLICY_URL})."
         nickname_info = textwrap.dedent(f"""
             Old nickname: `{old_nick}`
             New nickname: `{forced_nick}`
         """).strip()
 
+        formatted_reason = f'**Additional details:** {reason}\n\n' if reason else ''
+
+        embed_reason = (
+            f"Your previous nickname, **{old_nick}**, "
+            f"didn't comply with our nickname policy. "
+            f"Your new nickname will be **{forced_nick}**.\n\n"
+            f"{formatted_reason}"
+            f"You will be unable to change your nickname until **{expiry_str}**. "
```

Comment:
The expiry time is duplicated in two different spots in the direct message embed ([L115](https://github.com/python-discord/bot/blob/c3a927569da782c24299c8ae75df28ae6cd3f2ba/bot/exts/moderation/infraction/_scheduler.py#L115)). Can you remove this from the main embed, and only add it to the message sent to the invocation context?

Reply:
I think it's important to specify that they will be unable to change the nickname until the infraction expires, and in that case duplicating the date doesn't seem too bad.

Reply:
We can still mention that you can't change your name while this is in effect, without adding the date, but at that point, we're splitting hairs. If you're fine with the duplication, then I won't complain.